todo-client/src/utils/LocalStorage.js

```diff
@@ -1,13 +1,19 @@
-import { useState } from 'react'
+import { useEffect, useReducer } from 'react'
+import { TodoReducer } from '../reducers/TodoReducer'
 
-export const useLocalStorage = (name, initialTasks) => {
-   
-   const [localState] = useState(() => {
-    const tasks = JSON.parse(localStorage.getItem(name)) || initialTasks
-    return tasks
-   })
+export const getState = () => {
+    const state = JSON.parse(localStorage.getItem('app')) || []
```

Comment:
export const getState = () => JSON.parse(localStorage.getItem('app')) || [];

Don't need to define variable if you are using it just once